src/main/java/Yoda.java

```diff
@@ -0,0 +1,51 @@
+import yoda.YodaUI;
+import yoda.parser.Parser;
+import yoda.storage.Storage;
+import yoda.task.TaskList;
+import java.util.Scanner;
+import java.io.IOException;
+
+/**
+ * Yoda is a chatbot that helps users to manage their tasks.
+ * It can add, delete, and list tasks, as well as mark tasks as done.
+ * Yoda can also find tasks by searching for keywords.
+ */
+public class Yoda {
+    private final YodaUI yodaUI;
```

Comment:
```suggestion
    private final YodaUI YODA_UI;
```

constants should be uppercase with underscore to separate words.